lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -599,6 +599,23 @@ def _graded_scorable_blocks_to_header(cls, course):
 
 class ProblemResponses(object):
 
+    @staticmethod
+    def _build_block_base_path(block):
+        """
+        Return the display names of the blocks that lie above the supplied block in hierarchy.
+
+        Arguments:
+            block: a single block
+
+        Returns:
+            List[str]: a list of display names of blocks starting from the root block (Course)
+        """
+        path = []
+        while block.parent:
+            block = block.get_parent()
+            path.append(block.display_name)
+        return path[::-1]
```

Comment:
Also, `.display_name` is not guaranteed to exist for container types that exist below the level of Verticals. `LibraryContentModule` and `SplitTestModule` do define it, but there might be third party things that don't, so it might be worthwhile to have a fallback.